docs/references/dimensions.mdx

```diff
@@ -50,13 +50,12 @@ models:
         description: 'Total estimated revenue in GBP based on forecasting done by the finance team.'
         meta:
           dimension:
-            type: string
+            type: number
             label: 'Total revenue' # this is the label you'll see in Lightdash
             description: 'My custom description' # you can override the description you'll see in Lightdash here
             sql: 'IF(${TABLE}.revenue_gbp_total_est = NULL, 0, ${registered_user_email})' # custom SQL applied to the column from dbt used to define the dimension
             hidden: false
-            round: 2
-            format: 'gbp'
+            format: '[£]#,##0.00' # GBP rounded to two decimal points
```

Comment:
`[£]` is not valid. It should be `[$£]`
Currency is either `$` or `[$<text/symbol>]`